pydash/pydash_app/dashboard/endpoint.py

```diff
@@ -61,3 +61,97 @@ def aggregated_data(self):
         :return: A dict containing aggregated data points.
         """
         return self._aggregator_group.fetch_aggregator({}).as_dict()
+
+    def aggregated_data_daterange(self, start_date, end_date, granularity):
+        """
+        Returns the aggregated data on this dashboard over the specified daterange.
+        :param start_date: A datetime object that is treated as the inclusive lower bound of the daterange.
+        :param end_date: A datetime object that is treated as the inclusive upper bound of the daterange.
+        :param granularity: A string denoting the granularity of the daterange.
+        :return: A dictionary with all aggregated statistics and their values.
+        """
+        return self._aggregator_group.fetch_aggregator_inclusive_daterange({}, start_date, end_date,
+                                                                           granularity).as_dict()
+
+    def statistic(self, statistic, filters={}):
+        """
+        Returns the desired statistic of this dashboard, filtered by the specified filters.
+        :param statistic: A string denoting the specific statistic that should be queried.
+        """ \
+        f'  The following filters are allowed:{[stat_class.field_name(stat_class) for stat_class in Aggregator.contained_statistics_classes]}.' \
```

Comment:
you can also use `f""" ... """` to format docstrings :wink:

Reply:
True, but I am using `{` and `}` in a non-format way as well. This seemed like one of the most elegant looking solutions to me.

Reply:
Sure :+1: